rites/roman1969/src/locales/pt-br.ts

```diff
@@ -276,6 +290,7 @@ export const locale: Locale = {
       'São Dionísio, bispo e mártir, patrono da Cidade e da Diocese de Saint-Denis',
     divine_mercy_sunday: '2º Domingo do Tempo Pascal ou Domingo da Misericórdia',
     dominic_de_guzman_priest: 'São Domingos, presbítero',
+    dulce_lopes_pontes_virgin: 'Santa Dulce Lopes Pontes, virgem e religiosa', // src: Calendário Próprio do Brasil - CNBB
```

Comment:
a website or other source that we can leverage directly should be provided, those references need to be something we can verify